log4j-api/src/main/java/org/apache/logging/log4j/util/LoaderUtil.java

```diff
@@ -50,39 +50,31 @@ public final class LoaderUtil {
     // wants to use PropertiesUtil, but then PropertiesUtil wants to use LoaderUtil.
     private static Boolean ignoreTCCL;
 
-    private static final RuntimePermission GET_CLASS_LOADER = new RuntimePermission("getClassLoader");
-    private static final boolean GET_CLASS_LOADER_DISABLED;
-
-    private static final PrivilegedAction<ClassLoader> TCCL_GETTER = new ThreadContextClassLoaderGetter();
-
-    static {
-        if (System.getSecurityManager() != null) {
-            boolean getClassLoaderDisabled;
+    static final RuntimePermission GET_CLASS_LOADER = new RuntimePermission("getClassLoader");
```

Comment:
Unless I am mistaken, this is no longer necessary.

Reply:
It's used for checking if we have the ability to use the permission in full-permission mode to see if we should just fall back to the class loader of this class.